tests/ui/derive-invariant/helper-empty/expected

```diff
@@ -0,0 +1,6 @@
+error: Cannot derive impl for `PositivePoint`
+   |
+11 | #[derive(kani::Invariant)]
+   |          ^^^^^^^^^^^^^^^
+   |
+note: invariant condition in field `x` could not be parsed - `Err("expected attribute arguments in parentheses: #[invariant(...)]")`
```

Comment:
Can we remove the "Err" portion of the error? This should be:
```suggestion
note: invariant condition in field `x` could not be parsed: expected attribute arguments in parentheses: #[invariant(...)]
```

Reply:
Done.